hie-cabal.yaml

```diff
@@ -71,14 +108,14 @@ cradle:
   - path: ./iots-export/test
     config: { cradle: { cabal: { component: "iots-export:iots-export-test" } } }
 
-  - path: ./metatheory/src
-    config: { cradle: { cabal: { component: "lib:metatheory" } } }
-  - path: ./metatheory/test
-    config: { cradle: { cabal: { component: "metatheory:metatheory-test" } } }
+  - path: ./plutus-metatheory/src
+    config: { cradle: { cabal: { component: "lib:plutus-metatheory" } } }
+  - path: ./plutus-metatheory/test
+    config: { cradle: { cabal: { component: "plutus-metatheory:test1" } } }
```

Comment:
test1?

Reply:
https://github.com/input-output-hk/plutus/blob/master/plutus-metatheory/plutus-metatheory.cabal#L422

Reply:
This won't work perfectly because we have multiple components sharing the test dir... which I should talk to James about I guess!